### DISCUSSION TITLE
Convert to GoAccess format

### DISCUSSION THREAD

**Discussion by AllofTheThangz** [allinurl/goaccess#2211] [General]
Hello - a popular docker/container that uses NGINX uses this format:

log_format proxy '[$time_local] $status - $request_method $scheme $host "$request_uri" [Client $remote_addr] [Length $body_bytes_sent] [Gzip $gzip_ratio] "$http_user_agent" "$http_referer"'

This is for the official proxy hosts within NGINX Proxy Manager - Official

Can someone please convert this to goaccess format for the goaccess.conf?

**Comment by AllofTheThangz** [comment]
time-format %T
date-format %d/%b/%Y
log_format [%d:%t %^] %^ %^ %s - %m %^ %v "%U" [Client %h] [Length %b] [Gzip %^] [Sent-to %^] "%u" "%R"

I think...

**Comment by allinurl** [comment]
Please post a few sample lines directly from your access log. Thanks
